File: frontend/gestionale-web/src/app/models/cliente.ts
export interface Cliente {
  id: number;
  nome: string;
  cognome: string;
  email?: string | null;
  telefono?: string | null;
  societa?: string | null;
  note?: string | null;
}

/** Dati inviati alla creazione/aggiornamento (senza id). */
export type ClienteInput = Omit<Cliente, 'id'>;

File: backend/GestionaleApi/Models/Cliente.cs
namespace GestionaleApi.Models;

public class Cliente
{
    public int Id { get; set; }
    public string Nome { get; set; } = string.Empty;
    public string Cognome { get; set; } = string.Empty;
    public string CodiceFiscale { get; set; } = string.Empty;
    public string? PartitaIva { get; set; }
    public string? Pec { get; set; }
    public string? CodiceSdi { get; set; }
    public string? Email { get; set; }
    public string? Telefono { get; set; }
    public string? Societa { get; set; }
    public string? Note { get; set; }
}
